File: day068/homework/code/codewars3.js
// https://www.codewars.com/kata/5a3dd29055519e23ec000074/train/python
def check_exam(arr1, arr2):
    score = 0
    for correct, student in zip(arr1, arr2):
        if student == "":
            score += 0
        elif student == correct:
            score += 4
        else:
            score -= 1
    return max(score, 0)

File: day036/classwork/code/codewars3.py
# https://www.codewars.com/kata/65128732b5aff40032a3d8f0/train/python

def neutralise(s1, s2):
    res = ""
    for i in range(len(s1)):
        if s1[i] == s2[i]:
            res += s1[i]
        else:
            res += "0"
    return res
